communicator/room.go

```diff
@@ -3,17 +3,18 @@ package main
 import (
 	"fmt"
 	"log"
+	"math/rand"
 	"os"
 	"time"
 )
 
 const (
-	RoomSize       = 2
-	initialHp      = 100
-	initialMoney   = 5000
-	questionReward = 100
-	boardWidth     = 6
-	boardHeight    = 8
+	initialHp                = 100
+	initialMoney             = 5000
+	questionReward           = 100
+	boardWidth               = 6
+	boardHeight              = 8
+	battleRoyaleRoomCapacity = 8
```

Comment:
Shouldn't this constant be in hub.go? I'd say room should not know if it's in battle royale mode.